webpack.config.prod.js

```diff
@@ -29,6 +29,7 @@ export default {
     loaders: [
       {test: /\.js$/, include: path.join(__dirname, 'src'), loaders: ['babel']},
       {test: /\.(jpe?g|png|gif|svg)$/i, loaders: ['file']},
+      {test: /\.ico$/, loader: 'file-loader?name=[name].[ext]'},
```

Comment:
@jforaker @kwelch Looks like this is no longer needed since the master branch has been updated to webpack 2.